pkg/utils/util.go

```diff
@@ -129,6 +130,21 @@ func ValidateSnapshot(snapshot *crdv1.VolumeSnapshot) error {
 		return fmt.Errorf("VolumeSnapshot is nil")
 	}
 
+	vscname := snapshot.Spec.VolumeSnapshotClassName
```

Comment:
I think we should have a followup PR to move all these validating logic into webhook folder directly. having them defined here makes utils package depending on both v1 and v1beta1 which seems to be unnecessary. note that utils is used in controllers. let's create an issue to follow up on this?

Reply:
Sure